packages/nc-gui/components/virtual-cell/components/ListChildItems.vue

```diff
@@ -49,6 +49,8 @@ const {
   displayValueProp,
 } = useLTARStoreOrThrow()
 
+isChildrenLoading.value = true
```

Comment:
when will `isChildrenLoading` become `false`?

Reply:
```typescript
  const loadChildrenList = async () => {
      try {
        isChildrenLoading.value = true
        if (colOptions.value.type === 'bt') return
        if (!rowId.value || !column.value) return
        if (isPublic.value) {
          childrenList.value = await $api.public.dataNestedList(
            sharedView.value?.uuid as string,
            encodeURIComponent(rowId.value),
            colOptions.value.type as 'mm' | 'hm',
            column.value.id,
            {
              limit: String(childrenListPagination.size),
              offset: String(childrenListPagination.size * (childrenListPagination.page - 1)),
              where:
                childrenListPagination.query && `(${relatedTableDisplayValueProp.value},like,${childrenListPagination.query})`,
            } as any,
            {
              headers: {
                'xc-password': sharedViewPassword.value,
              },
            },
          )
        } else {
          childrenList.value = await $api.dbTableRow.nestedList(
            NOCO,
            (project?.value?.id || (sharedView.value?.view as any)?.project_id) as string,
            meta.value.id,
            encodeURIComponent(rowId.value),
            colOptions.value.type as 'mm' | 'hm',
            column?.value?.id,
            {
              limit: String(childrenListPagination.size),
              offset: String(childrenListPagination.size * (childrenListPagination.page - 1)),
              where:
                childrenListPagination.query && `(${relatedTableDisplayValueProp.value},like,${childrenListPagination.query})`,
            } as any,
          )
        }
        childrenList.value?.list.forEach((row: Record<string, any>, index: number) => {
          isChildrenListLinked.value[index] = true
          isChildrenListLoading.value[index] = false
        })
        if (!childrenListPagination.query) {
          childrenListCount.value = childrenList.value?.pageInfo.totalRows ?? 0
        }
      } catch (e: any) {
        message.error(`${t('msg.error.failedToLoadChildrenList')}: ${await extractSdkResponseErrorMsg(e)}`)
      } finally {
        isChildrenLoading.value = false
      }
    }
```


there is a final block here